packages/stylex/src/inject.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ */
+
+'use strict';
+
+import { createSheet } from './stylesheet/createSheet';
+import { addSpecificityLevel } from './stylesheet/utils';
+
+const sheet = createSheet();
+
+export default function inject(cssText: string, priority: number): void {
+  addSpecificityLevel(cssText, Math.floor(priority / 1000));
```

Comment:
This `priority/1000` logic probably needs to be revisited for both inject and the babel plugin. Can be done as part of a refactor of how priorities are defined and used.